drivers/1wire/1wire.c

```diff
@@ -45,6 +45,7 @@
 
 #include <nuttx/kmalloc.h>
 #include <nuttx/i2c/i2c_master.h>
```

Comment:
remove?

Reply:
That is the existing code not changed by me. I think kmalloc and i2c_master are not used here. There are a few dependencies to i2c (error/info) but this should not come from the i2c_master interface.

Reply:
Only i2c_master.h can be removed.